hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMAllocateBlockRequest.java

```diff
@@ -78,6 +80,29 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
     Preconditions.checkNotNull(allocateBlockRequest);
 
     KeyArgs keyArgs = allocateBlockRequest.getKeyArgs();
+    String keyPath = keyArgs.getKeyName();
```

Comment:
This functionality has duplicated over all the requests. Can you move this to OmClientRequest.java

`String OmClientRequest#validateAndNormalizeKey(BucketLayout bucketLayout, String keyPath, OzoneManager ozoneManager) throws OMException `

and we can use this method in the OM request classes.

`String keyPath = validateAndNormalizeKey(getBucketLayout(), keyArgs.getKeyName(), ozoneManager);`

Reply:
Done